### PR TITLE
Dp 7007 pano 2019

### PR DESCRIPTION
**Pull request by aditudorache**

### CODE REVIEW

File: src/map/components/legend/MapLegend.jsx [hunk 1]
```diff
@@ -29,6 +29,7 @@ class MapLegend extends React.Component {
         (mapLayer.layers && mapLayer.layers[0])}&`,
       'format=image/svg%2Bxml&',
       legendItem.imageSize ? `width=${legendItem.imageSize}&height=${legendItem.imageSize}&` : '',
```
Comment (royderks) [inline src/map/components/legend/MapLegend.jsx]: Rewrite to use the a stringified object

File: src/map/services/map-layers/map-layers.config.js [hunk 2]
```diff
@@ -75,10 +77,25 @@ const mapLayers = [
       mission_type: 'bi',
     },
   },
+  {
+    id: 'pano2019bi',
+    layers: ['panorama_new'],
+    legendItems: [{ title: '2019',
+    extraParameters: 'scale=800' }],
```
Comment (royderks) [inline src/map/services/map-layers/map-layers.config.js]: Use same notition as `params: { field: 'foo' }` to stringify in `MapLegend.jsx`

File: src/map/components/legend/MapLegend.jsx (hunk 1 above)
Comment (aditudorache) [reply src/map/components/legend/MapLegend.jsx]: good point!

File: src/map/services/map-layers/map-layers.config.js (hunk 2 above)
Comment (aditudorache) [reply src/map/services/map-layers/map-layers.config.js]: done!

**Pull request merged**